spec/components/key_pair_component_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe KeyPairGeneratorComponent, type: :component do
+  let(:options) do
+    {
+      location: 'component_spec',
+    }
+  end
+  subject(:rendered) do
+    render_inline(described_class.new(**options))
+  end
+
+  it 'logs an event' do
+    analytics = FakeAnalytics.new
+    allow(Analytics).to receive(:new).and_return(analytics)
+    allow(analytics).to receive(:track_event)
+
+    expect(analytics).
+      to have_received(:track_event).
+      with('Frontend: IdV: key pair generated')
+  end
```

Comment:
1. stubbing `.new` always jumps out me as a pattern to avoid if we can, because it's not spcific
2. where are we calling the `subject` in this spec?

Reply:
This file wasn't meant to be checked in as part of this PR and has been removed.

That said, I copied the following from prior art, e.g., https://github.com/18F/identity-idp/blob/main/spec/requests/rack_attack_spec.rb#L67-L69